apps/ai-content-generator/src/components/app/dialog/common-generator/output/output-text-panels/OutputTextPanels.tsx

```diff
@@ -38,7 +38,10 @@ const OutputTextPanels = (props: Props) => {
     const successfullyUpdated = await updateEntry(outputFieldId, outputFieldLocale, ai.output);
 
     if (successfullyUpdated) {
+      sdk.notifier.success('Content applied successfully.');
```

Comment:
These notification toasts had been previously added here: https://github.com/contentful/apps/blob/master/apps/ai-content-generator/src/hooks/dialog/useEntryAndContentType.ts#L66-L71, so we should just have them in one place.

Reply:
Ahh that's a good callout. I must have missed that! 

What are your thoughts on making this updateEntry return void and we just call it here?

Reply:
Yeah I'm fine with moving them here in the `OutputTextPanels` component, and then cleaning up what had been added in the `updateEntry` function!